### PR TITLE
[IOTDB-4067] Handle null pointer in getRegisteredNodeInfoList

### PR DESCRIPTION
**Pull request by MiniSho**
add handling of null pointer when NodeCacheMap doesn't find the NodeId, and set the status as  "Unknown".

### CODE REVIEW

**Review by CRZbulabula — COMMENTED** [review]
Good job! Please also check whether the interfaces of show cluster, show confignodes, show datanodes and show regions has this hidden danger~

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java [hunk 1]
```diff
@@ -232,8 +234,12 @@ public List<TDataNodeInfo> getRegisteredDataNodeInfoList() {
             TDataNodeInfo info = new TDataNodeInfo();
             int dataNodeId = dataNodeInfo.getLocation().getDataNodeId();
             info.setDataNodeId(dataNodeId);
-            info.setStatus(
-                getLoadManager().getNodeCacheMap().get(dataNodeId).getNodeStatus().getStatus());
+            INodeCache nodeCache = getLoadManager().getNodeCacheMap().get(dataNodeId);
```
[the remark this answers is not in the document]
Comment (MiniSho) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java]: done

**Review by Beyyes — APPROVED** [review]

**Comment by MiniSho** [comment]
already checked showCluster, showConfigNodes, showDataNodes and showRegion, and didn't find any hidden danger

**Review by CRZbulabula — APPROVED** [review]
LGTM～

**Pull request merged**